### ISSUE TITLE
the result

### ISSUE THREAD

**Issue by vicdu** [bearpaw/pytorch-pose#42]
The result in the paper is 90.09, but the recurring result is 88.78, where do you think the problem occurred?

**Comment by heroxx2011** [comment]
Hi vidcu, I have trained the model and the result only 84.89. I think maybe the parameters problem. Welcome to discuss.

**Comment by vicdu** [comment]
@heroxx2011 hi, Have you read the code in torch7 ? I found something like '+offset' in dataload part. And, do you think it will be beneficial to expand the mpi-dataset?

**Comment by mkocabas** [comment]
Hi @vicdu,
The result of the original code is on the MPII test set. In general, test accuracy is ~2 point higher than validation accuracy. @bearpaw's implementation give better accuracy than original one. You can take a look at these discussions:
https://github.com/umich-vl/pose-hg-demo/issues/1 
https://github.com/umich-vl/pose-hg-demo/issues/6
https://github.com/bearpaw/pytorch-pose/issues/15
